rskj-core/src/test/java/co/rsk/test/builders/P2shErpFederationBuilder.java

```diff
@@ -0,0 +1,79 @@
+package co.rsk.test.builders;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.NetworkParameters;
+import co.rsk.peg.bitcoin.BitcoinTestUtils;
+import co.rsk.peg.federation.ErpFederation;
+import co.rsk.peg.federation.FederationArgs;
+import co.rsk.peg.federation.FederationFactory;
+import co.rsk.peg.federation.FederationMember;
+import co.rsk.peg.federation.constants.FederationConstants;
+
+import java.time.Instant;
+import java.util.List;
+
+public class P2shErpFederationBuilder {
+    private List<BtcECKey> membersPublicKeys;
```

Comment:
```suggestion
    private List<FederationMember> members;
```
This will give more flexibility to have same or different keys for btc/rsk/mst

Reply:
Another option

```suggestion
    private List<BtcECKey> membersBtcPublicKeys;
    private List<BtcECKey> membersRskPublicKeys;
    private List<BtcECKey> membersMstPublicKeys;
```
Then in the constructor, `membersRskPublicKeys` and `membersMstPublicKeys` are initialized as empty list.

Finally in `build` method, if not empty you build the FederationMembers using all 3 list. If empty, build FederationMembers using Btc keys for all 3.

Thoughts?

Reply:
i like the idea!
thanks